core/edit/edit_command_actions.py

```diff
@@ -88,26 +88,37 @@ def edit_action(m) -> EditAction:
 }
 
 
-def run_action_callback(action: EditAction):
-    action_type = action.type
-
-    if action_type in simple_action_callbacks:
-        callback = simple_action_callbacks[action_type]
-        callback()
-        return
-
-    match action_type:
-        case "insert":
-            assert isinstance(action, EditInsertAction)
-            actions.insert(action.text)
-
-        case "wrapWithDelimiterPair":
-            assert isinstance(action, EditWrapAction)
-            return lambda: actions.user.delimiter_pair_wrap_selection(action.pair)
-
-        case "applyFormatter":
-            assert isinstance(action, EditFormatAction)
-            actions.user.formatters_reformat_selection(action.formatters)
-
-        case _:
-            raise ValueError(f"Unknown edit action: {action_type}")
+@mod.action_class
+class Actions:
+    def run_action_callback(action: EditAction):
```

Comment:
Can you add the word edit into the name of this action somewhere? It is a little too generic otherwise.